routes/web/hardware.php

```diff
@@ -146,9 +146,9 @@ function () {
             [AssetFilesController::class, 'show']
         )->name('show/assetfile');
 
-        Route::delete('{assetId}/showfile/{fileId}/delete',
-            [AssetFilesController::class, 'destroy']
-        )->name('delete/assetfile');
+        Route::delete('{assetId}/delete',
+            [AssetsController::class, 'destroy']
+        )->name('hardware/delete');
```

Comment:
This route already exists - we shouldn't have to add or remove anything from routes for this.

Reply:
Oh, it was removed on my side and I had added it back. Sorry got crossed up.